draft-ietf-mandyam-eat.md

```diff
@@ -260,6 +260,111 @@ signing. In this case the manufacturer should perform the verification,
+themselves, not just of the signing format and representation syntax.
+
+This is a lofty goal and it will be difficult to achieve broadly as it
+requires careful definition of claims in a technology neutral way.
+Sometimes two entity types will have two ways of doing something that
+are largely irreconcilable. However, the goal remains, even if it is a
+very long-term goal, where entities gradually get designed so that
+they more and more easily can interoperate with EAT.
+
+### OS and Technology Neutral
+
+Claims should be defined such that they are not specific 
+to any operating system. They should be applicable to
+multiple large high-level operating systems from different
+vendors. They should also be applicable to multiple 
+embedded operating systems from multiple vendors and
+everything in between.
+
+Claims should not be defined such that they are specific to any SW
+environment or programming language.
+
+Claims should not be defined such they are not specific to any chip or
+hardware. For example, they should not just be the contents of some HW
+status register as it is unlikely that the same HW status register
+with the same bits exists on a chip of a different manufacturer.
+
+The boot and debug state claims in this document are an example of a
+claim that has been defined in this neutral way.
+
+### Security Level Neutral
+
+Many use cases will have tokens generated by some of the most secure
+hardware and software that exists.  Secure elements and smart cards
+are examples of this.  However, EAT is intended for use in low
+security use cases just the same. For example, an app in a mobile app
+download system may generate tokens.
+
+The claims included in this document are on the basis of whether they
+are useful and interoperable.  No claims have been excluded because
+they are not useful on high-security environments and entities.
+
+### Reuse of Extant Data Formats
+
+Where possible, claims should already standardized data items,
+identifier and data formats. This takes advantage of the expertise in
+creating those formats and improves interoperability.
+
+In some cases, a proprietary data format that is already in use may be
+adopted and made standard for a claim. It should fulfill the above
+criteria, particular the OS and technology neutrality.
+
+Often these claims will not be defined in the encodings, the syntaxes,
+used by EAT. In most cases it is best to wrap the data or adapt the
+encoding to the ones used by EAT so that EAT implementations do not
+end up with a plethora of encoders and decoders of different
+serialization formats.
+
+In some rare cases, it may be better to use the encoding as is. For
+example, signed X.509 certificates and CRLS should be carried as is
+and not re encoded.
+
+### Proprietary Claims
+
+General compliance with EAT allows arbitrary proprietary claims 
+to be added. In many cases proprietary claims will be the
+easiest and most obvious way to proceed. 
```

Comment:
Add comment to the effect that proprietary claims should not replace or overlap standard claims unless the implementation is significantly different?